bin/dotc

```diff
@@ -20,7 +20,7 @@ function getLastStringOnLineWith {
 # Configuration
 SCALA_VERSION=$(getLastStringOnLineWith "scalaVersion in")
 SCALA_BINARY_VERSION=2.11
-SCALA_COMPILER_VERSION=$(getLastStringOnLineWith "scala-compiler")
+SCALA_COMPILER_VERSION=$(getLastStringOnLineWith "scalaCompilerVersion")
```

Comment:
This now returns two lines:
```scala
$ grep "scalaCompilerVersion" "project/Build.scala"|sed -n 's/.*\"\(.*\)\".*/\1/'p
2.11.5-20151022-113908-7fb0e653fd
scala-compiler
```
You can avoid this by doing `getLastStringOnLineWith "def scalaCompilerVersion"`, optionally,  you could also change the definition of `getLastStringOnLineWith` so that it only returns one line:
```bash
grep "$PATTERN" "$DOTTY_ROOT/project/Build.scala"|sed -n 's/.*\"\(.*\)\".*/\1/'p|head -n1
```